src/app/content/studyGuides/components/UsingThisGuide/UsingThisGuideButton.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled, { css } from 'styled-components/macro';
+import { PlainButton } from '../../../../components/Button';
+import theme from '../../../../theme';
+import { toolbarDefaultText } from '../../../components/Toolbar/styled';
+import { QuestionCircle } from 'styled-icons/fa-regular/QuestionCircle';
+import { mobilePaddingSides, barHeight } from '../../../styles/PopupConstants';
+
+export const UTGButton = {
+  height: barHeight,
+  marginBottom: 0.6,
+  negativeMarginBottom: 0.1
+}
+
+// tslint:disable-next-line:variable-name
+const QuestionIcon = styled(QuestionCircle)`
+  height: 1.6rem;
+  width: 1.6rem;
+`;
+
+// tslint:disable-next-line:variable-name
+export const UsingThisGuideButtonWrapper = styled(PlainButton)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: auto;
+  position: relative;
+  overflow: visible;
+  color: ${(props) => props.isOpen ? theme.color.white : theme.color.primary.gray.base};
+  background: ${(props) => props.isOpen ? 'black' : 'white'};
+  padding:  ${(props) => props.isOpen ? '0 1.6rem' : '0'};
+  height: ${UTGButton.height - UTGButton.marginBottom}rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const UsingThisGuideText = styled.span`
+  ${toolbarDefaultText}; 
+`;
+
+// tslint:disable-next-line:variable-name
+const UsingThisGuideWrapper = styled.div`
+  margin-right: 4.8rem;
+  margin-bottom: ${(props) => props.isOpen ? -(UTGButton.marginBottom + UTGButton.negativeMarginBottom) : '0'}rem;
```

Comment:
The button "jumps" a little because of the changing margin. I think you should rather aim for making the button longer vertically while keeping the contents in the same place rather than pushing it down.

Reply:
I put the negative margin because even putting the button longer, it does not cover the `border-bottom` from the filters bar, so the black background on the button never reaches the banner, can you take a look into that?

Reply:
Maybe you could try with adding a missing background with `:after` selector